### ISSUE TITLE
Exception skip_checks is not a valid argument for this message type after upgrading to 1.3.1

### ISSUE THREAD

**Issue by mehdilauters** [mido/mido#570]
Hello, please find an issue for your last release.
We will keep the previous version for now.
Thanks for your work

**Describe the bug**
the message loop now raise an exception which make the library unable to parse our midi files with the following trace:
```python-traceback
 - in file /opt/magichanism/scenarios/rooms/lazers/lazers_level.py:40, in run : for msg in mid:
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/midifiles.py:391, in __iter__ : for msg in self.merged_track:
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/midifiles.py:330, in merged_track : self._merged_track = merge_tracks(self.tracks, skip_checks=True)
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/tracks.py:118, in merge_tracks : messages.extend(_to_abstime(track, skip_checks=skip_checks))
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/tracks.py:72, in _to_abstime : yield msg.copy(skip_checks=skip_checks, time=now)
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/meta.py:518, in copy : return self.__class__(**attrs)
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/meta.py:488, in __init__ : raise ValueError(
Exception ValueError in thread MidiP.safe_run: skip_checks is not a valid argument for this message type
 - in file /usr/share/mgi/mgi-game-server/misc/threadpool.py:50, in run : self.safe_run()
 - in file /usr/share/mgi/mgi-game-server/game/scheduler.py:44, in safe_run : self.callback(self, self.ask_to_quit)
 - in file /opt/magichanism/scenarios/rooms/lazers/lazers_level.py:24, in safe_run : self.run(_, ask_to_quit)
 - in file /opt/magichanism/scenarios/rooms/lazers/lazers_level.py:40, in run : for msg in mid:
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/midifiles.py:391, in __iter__ : for msg in self.merged_track:
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/midifiles.py:330, in merged_track : self._merged_track = merge_tracks(self.tracks, skip_checks=True)
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/tracks.py:118, in merge_tracks : messages.extend(_to_abstime(track, skip_checks=skip_checks))
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/tracks.py:72, in _to_abstime : yield msg.copy(skip_checks=skip_checks, time=now)
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/meta.py:518, in copy : return self.__class__(**attrs)
 - in file /usr/local/lib/python3.8/dist-packages/mido/midifiles/meta.py:488, in __init__ : raise ValueError(
Exception ValueError in thread MidiP.safe_run: skip_checks is not a valid argument for this message type
```

**Code Snippet**
Pseudo code extracted from our sources:
```python
class MidiPlayer:
    def __init__(self, midifile, lazers):
        self.midifile = midifile
        self.run()

    def run(self, _, ask_to_quit):
        while not ask_to_quit.is_set():
            mid = mido.MidiFile(self.midifile)
            for msg in mid:
                t += msg.time
                try:
                    if msg.type == 'note_on':
                    elif msg.type == 'note_off':
                except Exception as e:
                    log_error("midi exception %s" % e)

    def note_to_addr(self, note):
        return 127-note
````

**Context (please complete the following information):**
- Operating system name & version: 
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.6 LTS
Release:	20.04
Codename:	focal

- Python version: Python 3.8
- mido package version 1.3.1 installed from PyPI
- backend used (amidi, portmidi, rtmidi, PyGame…): any

**Additional context**
Please find an example of our midi file
[level_1.1.zip](https://github.com/mido/mido/files/13684788/level_1.1.zip)

**Comment by rdoursenaud** [comment]
Thanks for the detailed report.

I'll investigate as soon as possible.

At a glance, it looks like a side effect of #474 that focused on midi message events and missed handling meta events.
@almostimplemented would you mind looking into it?

Apologies for the inconvenience.

**Comment by rdoursenaud** [comment]
Should be fixed in the just released 1.3.2 thanks to @almostimplemented.

Thanks again!
